.scrutinizer.yml

```diff
@@ -2,9 +2,10 @@ before_commands:
     - "composer update --prefer-source"
 
 tools:
+    external_code_coverage:
+        timeout: 1800
     php_code_coverage:
         enabled: true
-        test_command: ./vendor/bin/phpunit
```

Comment:
In case of multiple coverage files, there is `runs:  2` directive, to merge them.

https://scrutinizer-ci.com/docs/tools/external-code-coverage/#merging-coverage-data